jvmhs/src/Jvmhs/Data/Code.hs

```diff
@@ -158,9 +166,551 @@ verificationTypeInfo g (B.StackMapTable s) =
 $(deriveToJSON defaultOptions{fieldLabelModifier = camelTo2 '_' . drop 5} ''Code)
 $(deriveToJSON defaultOptions{fieldLabelModifier = camelTo2 '_' . drop 3} ''ExceptionHandler)
 
-instance ToJSON (B.ByteCodeOpr B.High) where
-  toJSON _ = String "<code>"
+instance ToJSON ByteCodeOpr where
+  toJSON bopr =
+    object $ ("opc" .= byteCodeOprOpCode bopr) : case bopr of
+
+    B.ArrayLoad arrType ->
+      [ "type" .= fromArrayType arrType
+      ]
+
+    B.ArrayStore arrType ->
+      [ "type" .= fromArrayType arrType
+      ]
+
+    B.Push bconstant ->
+      getListOfPairsFromBConstant bconstant
+
+    B.Load localtype localaddr ->
+      [ "type" .= fromLocalType localtype
+      , "addr" .= localaddr
+      ]
+
+    B.Store localtype localaddr ->
+      [ "type" .= fromLocalType localtype
+      , "addr" .= localaddr
+      ]
+
+    B.BinaryOpr binopr arithmetictype ->
+      [ "opr" .= binopr
+      , "type" .= fromArithmeticType arithmetictype
+      ]
+
+    B.Neg arithmetictype ->
+      [ "type" .= fromArithmeticType arithmetictype
+      ]
+
+    B.BitOpr bitopr wordsize ->
+      [ "opr" .= bitopr
+      , "size" .= wordsize
+      ]
+
+    B.IncrLocal localaddr increment ->
+      [ "addr" .= localaddr
+      , "incr" .= increment
+      ]
+
+    B.Cast castopr ->
+      [ "opr" .= castopr
+      ]
+
+    B.CompareLongs -> []
+
+    B.CompareFloating bool wordsize ->
+      [ "gt" .= bool
+      , "size" .= wordsize
+      ]
+
+    B.If cmpOpr oneOrTwo shortRelativeRef ->
+      [ "opr" .= cmpOpr
+      , "operands" .= oneOrTwo
+      , "offset" .= shortRelativeRef
+      ]
+
+    B.IfRef bool oneOrTwo shortRelativeRef ->
+      [ "equal" .= bool
+      , "operands" .= oneOrTwo
+      , "offset" .= shortRelativeRef
+      ]
+
+    B.Goto longrelativeref ->
+      [ "target" .= longrelativeref
+      ]
+
+    B.Jsr longrelativeref ->
+      [ "target" .= longrelativeref
+      ]
+
+    B.Ret localaddr ->
+      [ "addr" .= localaddr
+      ]
+
+    B.TableSwitch longrelativeref switchtable ->
+      [ "ref" .= longrelativeref
```

Comment:
This ref have some meaning

Reply:
```suggestion
      [ "default" .= longrelativeref
```
https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-6.html#jvms-6.5.tableswitch